ironic/tests/unit/drivers/modules/drac/test_management.py

```diff
@@ -821,3 +827,99 @@ def test_clear_job_queue(self, mock_get_drac_client):
                 job_ids=['JID_CLEARALL_FORCE'])
 
             self.assertIsNone(return_value)
+
+@mock.patch.object(drac_mgmt, 'redfish_utils', autospec=True)
```

Comment:
So I think we want to add the following here:
@mock.patch.object(drac_common, 'get_sushy_oem_manager', spec_set=True,
                   autospec=True)

Reply:
Addressed.